src/OrbitQt/AccessibilityAdapter.cpp

```diff
@@ -205,14 +205,25 @@ QAccessibleInterface* AccessibilityAdapter::childAt(int x, int y) const {
 }
 
 QRect AccessibilityAdapter::rect() const {
-  orbit_accessibility::AccessibilityRect rect = info_->AccessibleLocalRect();
-  if (parent() == nullptr) {
-    return QRect(rect.left, rect.top, rect.width, rect.height);
+  orbit_accessibility::AccessibilityRect rect = info_->AccessibleRect();
+
+  // Find the outer most AccessibleInterface to convert the coordinates from
+  // pixel relative to the CaptureWindow to screen positions.
+
+  const orbit_accessibility::AccessibleInterface* bridge_info = info_->AccessibleParent();
+  const QAccessibleInterface* bridge = parent();
+  while (bridge_info != nullptr && bridge_info->AccessibleParent() != nullptr) {
+    bridge_info = bridge_info->AccessibleParent();
+    bridge = bridge->parent();
+  }
```

Comment:
This could be easier to read if you only trace `AccessibleParent()` upwards, and rename `bridge_info` to `root` or similar.

Once `root->AccessibleParent() == nullptr`, you should be able to retrieve `QAccessibleInterface` from `AdapterRegistry::GetOrCreateAdapter`.

At least consider naming the variables differently so it's clear you're searching for the root of our GL accessibility tree here, I think "bridge" doesn't really help to understand this.

Reply:
Done.